git-gud.1

```diff
@@ -0,0 +1,68 @@
+.TH git-gud 1 "19 Feb 2020" "version 0.0"
+.SH NAME
+git-gud \- get good at git with git-gud
+.SH SYNOPSIS
+.B git gud
+[option1] [option2]
+.SH DESCRIPTION
+.B git-gud
+teaches you how to use Git with fun interactive tutorials.
```

Comment:
"Teaches..."

Reply:
Again, this is supposed to be lowercase. If you view the man page, you will see this.

Reply:
I viewed the man page, and I see what you mean.

Instead, can you change the bolded "git-gud" to unbolded "Git Gud"? The official name is "Git Gud" and the command is "git gud" just like "Git"'s command is "git"

Reply:
This is true; however, the package name is git-gud. So, I figured that would go under name. For example, URxvt (which has a man page that is accessed with `man urxvt`) comes from the package `rxvt-unicode`, and the latter is what is under `NAME` in the man page. I think this makes things easier (at least, for myself it does); however, I can understand your point.